waspc/data/Generator/templates/server/src/auth/hooks.ts

```diff
@@ -0,0 +1,39 @@
+{{={= =}=}}
+import { prisma } from 'wasp/server'
+import { type ProviderId } from 'wasp/auth/utils'
+import { type AuthHookFn } from 'wasp/auth/hooks'
+
+{=# onUserSignUpHook.isDefined =}
+{=& onUserSignUpHook.importStatement =}
+export const onUserSignUpHook = addExtraParamsToTheHookCall({= onUserSignUpHook.importIdentifier =})
+{=/ onUserSignUpHook.isDefined =}
+{=^ onUserSignUpHook.isDefined =}
+export const onUserSignUpHook = undefined
+{=/ onUserSignUpHook.isDefined =}
+
+{=# onUserLoginHook.isDefined =}
+{=& onUserLoginHook.importStatement =}
+export const onUserLoginHook = addExtraParamsToTheHookCall({= onUserLoginHook.importIdentifier =})
+{=/ onUserLoginHook.isDefined =}
+{=^ onUserLoginHook.isDefined =}
+export const onUserLoginHook = undefined
+{=/ onUserLoginHook.isDefined =}
+
+{=# onUserLogoutHook.isDefined =}
+{=& onUserLogoutHook.importStatement =}
+export const onUserLogoutHook = addExtraParamsToTheHookCall({= onUserLogoutHook.importIdentifier =})
+{=/ onUserLogoutHook.isDefined =}
+{=^ onUserLogoutHook.isDefined =}
+export const onUserLogout = undefined
```

Comment:
```suggestion
export const onUserLogoutHook = undefined
```

Is `Hook` missing here? I added it above.